packages/ui/package.json

```diff
@@ -35,6 +35,7 @@
     "@enact/core": "^5.0.0-alpha.1",
     "@enact/i18n": "^5.0.0-alpha.1",
     "classnames": "^2.5.1",
+    "ilib": "14.19.0",
```

Comment:
this was needed in order to solve this error:
![image](https://github.com/user-attachments/assets/fce72013-e255-46a6-81fb-0435c5038dcf)